### PR TITLE
WooCommerce HPOS Issue - Added filter to extend WooCommerce URL.

### PR DESCRIPTION
**Pull request by redscar**
Ticket: [ETP-901]

Added a new filter so that WooCommerce can hook into and add the proper URL.

https://github.com/the-events-calendar/event-tickets-plus/pull/1519

[ETP-901]: https://stellarwp.atlassian.net/browse/ETP-901?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Tribe/Attendees_Table.php [hunk 1]
```diff
@@ -307,10 +307,23 @@ public function column_status( array $item ) {
...
+		/**
+		 * Filter the order ID URL.
+		 *
+		 * @since TBD
+		 *
+		 * @param string $order_id_url The URL of the order ID.
+		 * @param array  $item The item array.
+		 */
+		return apply_filters(
+			'tec_attendees_table_order_id_url',
```
Comment (bordoni) [inline src/Tribe/Attendees_Table.php:323]: `tec_tickets_` is the correct prefix.

File: src/Tribe/Attendees_Table.php [hunk 2]
```diff
@@ -307,10 +307,23 @@ public function column_status( array $item ) {
 	public function get_order_id_url( array $item ) {
 		// Backwards compatibility.
 		if ( empty( $item['order_id_url'] ) ) {
+			// @todo fix this piece
```
Comment (bordoni) [inline src/Tribe/Attendees_Table.php:310]: What is this todo?

**Review by bordoni — CHANGES_REQUESTED** [review]
Changelog?

**Review by codingmusician — APPROVED** [review]

**Comment by rafsuntaskin** [comment]
This filter is not needed, we should include the data within the WooCommerce Provider `get_attendee` method as we do for the [EDD](https://github.com/the-events-calendar/event-tickets-plus/blob/fc988a79ede9ceb0095b370ac4b8185fd51c00b4/src/Tribe/Commerce/EDD/Main.php#L1595).

**Review by rafsuntaskin — CHANGES_REQUESTED** [review]
This filter is not needed, we should include the data within the WooCommerce Provider `get_attendee` method as we do for the [EDD](https://github.com/the-events-calendar/event-tickets-plus/blob/fc988a79ede9ceb0095b370ac4b8185fd51c00b4/src/Tribe/Commerce/EDD/Main.php#L1595).

**Comment by redscar** [comment]
Closing this ticket due to it not being needed for the Woocommerce work. 

https://github.com/the-events-calendar/event-tickets-plus/pull/1519/files#diff-cb640b1c62c1053fc7d6483c39896d9d7559ed05cb62e2585558aaf422ee936bR2507

**Pull request closed without merging**